insights/terraform/projects/bb2/README.md

```diff
@@ -12,43 +12,65 @@ The BB2 specific components are setup under the following service areas:
 
 - `services/analytics`:
   - Includes components for the lambda and event scheduler that update the intermediate reporting tables via Athena and for usage in QuickSight.
+    - For more details, see this module [README.md](services/analytics/modules/lambda/update_athena_metric_tables/README.md).
+    - Is updated when adding new metrics (except calculated fields).
```

Comment:
@dtisza1  Hmm not clear on line 16. Are you saying that this section of code should be modified when adding new metrics?

Reply:
> @dtisza1 Hmm not clear on line 16. Are you saying that this section of code should be modified when adding new metrics?

@meliGuzman  Was trying to denote which of the service areas are related to adding new metrics.  Maybe should remove that from here, since it is documented elsewhere. For example, the lambda templates need updated under that area for new metrics.  Does removing or rewording sound good to you?

Reply:
@dtisza1 i think either or works so maybe reword this to "when adding new metrics except for calculated field services/analytics would need to be updated"